tests/all_tests_spec.yaml

```diff
@@ -0,0 +1,132 @@
+  - name: fw_rules_csv_md
+    type: scheme
+    root: fw_rules_tests/policies/
+    expected_output: ./fw_rules_tests/policies/expected_output/
+    files_list:
+      - poc1-scheme.yaml
+      - test4-scheme.yaml
+      - semantic_diff_computeDiff-scheme.yaml
+      - semantic_diff_namedPorts-scheme.yaml
+      - semantic_diff_with_different_topologies-scheme.yaml
+      - semantic_diff_withIpBlock-scheme.yaml
+    out_format_arg: ['csv', 'md', 'dot']
+    add_out_path_arg: true
+    out_path_arg_suffix: '_output'
+
+  #3: use outputConfig from scheme files (don't override) => (multiple output files + stdout , multiple output formats)
+  # applies to all scheme files in tests/output_config_tests/policies/
+  - name : output_config_no_cli_args
+    type: scheme
+    root: output_config_tests/policies/
+    expected_output: ./output_config_tests/expected_output_files/
+    add_out_path_arg: false
+
+
+  #4: override only out file arg (use outputConfig from scheme files for out format) => (one output file per test, multiple output formats within this file content)
+  # applies to all scheme files in tests/output_config_tests/policies/
+  - name: output_config_only_out_file_arg_overriden
+    type: scheme
+    root: output_config_tests/policies/
+    expected_output: ./output_config_tests/expected_output_files/
+    add_out_path_arg: true
+    out_path_arg_suffix: '_override_only_out_file_arg_output'
+
+  #5: override only out format arg (use outputConfig from scheme files for out file / stdout) => (multiple output files per test, one output format)
+  # applies to all scheme files in tests/output_config_tests/override_out_format_only/
+  - name: output_config_only_out_format_arg_overriden
+    type: scheme
+    root: output_config_tests/override_out_format_only/
+    expected_output: ./output_config_tests/expected_output_files/
+    out_format_arg: ['txt']
+
+
+  # 6: use queries from cmdline tests (connectivity/semantic-diff) [no scheme-file, thus no outputConfig]
+  # out_format and out_path as defined on each cli query args => (one output file or stdout per query, one output format)
+  # covers various combinations : (no output args , only out_format arg, only out_path arg, both out_format and out_path args)
+  # applies to cli queries in tests/output_config_tests/output_cmdline_tests.yaml
+  - name: output_cmdline_tests
+    type: cmdline
+    root: output_config_tests/
+    expected_output: ./output_config_tests/expected_output_files/
+    files_list:
+      - output_cmdline_tests.yaml
+
+#---------------------------------------------------------------------------------------------------------------------------------
+
+# fw_rules_assertions: activate assertions to verify equivalence of  fw_rules output (semantically) to the input connectivity graph.
+fw_rules_assertions:
+  # 1:  all scheme files under tests/fw_rules_tests/policies/
+  - name: all_scheme_tests
+    type: scheme
+    root: fw_rules_tests/policies/
```

Comment:
Add a flag that activates fw-rules assertion (rather than relying on category name)

Reply:
done